src/main/java/edu/asu/plp/tool/backend/isa/events/AssemblerResultEvent.java

```diff
@@ -2,13 +2,24 @@
 
 import edu.asu.plp.tool.backend.isa.ASMImage;
 
+/**
+ ** This class is controls the assembler and manages the result
+ **/
 public class AssemblerResultEvent {
 
 	private boolean assembleSuccess;
 	private String errorMessage;
 	private String projectName;
 	private ASMImage asmImage;
-	
+
+	/**
+	 * This is a constructor for which accepts Success, ErrorMessage, ProjectName and ASMImage parameters
+	 * @param {String} Success        - It takes the assembler success parameter
+	 * @param {String} ErrorMessage   - It takes the error message parameter
+	 * @param {string} ProjectName    - It takes the name of the project
+	 * @param {string} ASMImage       - It takes the ASM Image
+	 * **/
+
```

Comment:
Please add functions of parameters for example success - possible values are true which means assembled successfully and false means could not assemble or ProjectName indicate the project name for which the assembler result event generated